### DISCUSSION TITLE
Can I configure ip address to be US-based?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#16171] [Questions and Help] [answered]
Hi,

Certain features on our websites are available to US-based ip addresses only. Is it possible to set ip address to US region in Cypress?

Thanks.

**Comment by bahmutov** [comment, marked as answer]
Cypress does not control the IP address, it just runs in whatever container you start it in. Thus if you want to change the IP during the test, it is something you should configure in your CI setup.

**Comment by aliabbasmalik8** [comment]
I want to run the e2e tests each time with a different proxy. Is that possible in cypress?
